Automattic/jetpack
# Payments Block: Propagate shared component controls to children

Fixes #23565

#### Changes proposed in this Pull Request:

* Added support to propagate the Product Management Controls to child blocks.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create or edit a new post
* Add a Payment Button block.
* You should see the Product Management Controls in every InnerBlock of the Payment Button block.
